src/test/java/uk/gov/hmcts/reform/sendletter/services/FtpUploadTest.java

```diff
@@ -9,21 +9,22 @@
 import java.io.File;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.sendletter.LocalSftpServer.port;
 
 public class FtpUploadTest {
 
     @Test
     public void connects_to_ftp() throws Exception {
-        try (LocalSftpServer server = LocalSftpServer.create()) {
-            FtpHelper.getClient(server.port).testConnection();
-        }
+        LocalSftpServer server = LocalSftpServer.create();
+        FtpHelper.getSuccessfulClient(port).testConnection();
+        server.close();
```

Comment:
why not try-with-resource?

Reply:
there's nothing to try here plus `server` would be unused code

Reply:
But then if there's an exception in testConnection the server will not be closed.

Reply:
checkstyle and pmd doesn't give warning for unused code - rolling back 👍